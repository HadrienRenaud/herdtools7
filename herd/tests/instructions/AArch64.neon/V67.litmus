AArch64 V67
{
  uint8_t x[16];
  0 : X0 = x;
}
P0;
MOVI V0.16B,#1        ; (* V0 = {1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1} *)
MOVI V1.16B,#2        ; (* V1 = {2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2} *)
EOR V2.8B,V1.8B,V0.8B ; (* V2 = {3,3,3,3,3,3,3,3,0,0,0,0,0,0,0,0} *)
ST1{V2.16B},[X0]      ;

forall x={3,3,3,3,3,3,3,3,0,0,0,0,0,0,0,0}
