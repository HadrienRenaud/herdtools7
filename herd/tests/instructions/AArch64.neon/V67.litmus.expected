Test V67 Required
States 1
x={3,3,3,3,3,3,3,3,0,0,0,0,0,0,0,0};
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (x={3,3,3,3,3,3,3,3,0,0,0,0,0,0,0,0})
Observation V67 Always 1 0
Hash=24bfe8345767d32cf5f5aca4f56af385

